### ISSUE TITLE
v0.13 release is bust

### ISSUE THREAD

**Issue by jnikula** [jnikula/hawkmoth#148]
The packaging is broken. See #147.

```
Extension error:
Could not import extension hawkmoth.ext.transformations (exception: No module named 'hawkmoth.ext')

```

**Comment by jnikula** [comment]
After quick fixes, need to a) start testing the packaged version, and b) use wildcards in setup.cfg to include all the packages.

**Comment by jnikula** [comment]
Fixed by 9ad923e5cf0f ("build: actually package hawkmoth.ext.*") and released in v0.14.0.
